src/workers/talk-with-ai-worker.ts

```
@@ -72,313 +92,483 @@ const postMessage = (messages: Message, options?: WindowPostMessageOptions) => {
-    const sts = await SpeechToSpeech.getInstance();
-    sts.setProgressCallback(modelType, (info) => {
-        postMessage({
-            type: "INIT",
-            status: info.status,
-            data: info,
-        });
-    });
     switch (modelType) {
         case "STT":
-            await sts.getSpeechToTextGenerator();
+            const stt = await SpeechToText.getInstance();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)